src/embed/VbrickEmbedConfig.ts

```diff
@@ -54,6 +55,33 @@ export interface VbrickVideoEmbedConfig extends VbrickBaseEmbedConfig {
 	hideSettings?: boolean;
 	hideFullscreen?: boolean;
 
+	/**
+	 * shows video player with sidebar.
+	 */
+	fullPlayer?: boolean;
+
+	/**
+	 * provide the sidebar name to be opened by default when video loads.
+	 * enableFullVodPlaybackInEmbed value needs to be true for this to work.
+	 */
+	defaultSidebar?: VideoPlaybackSidebarButton;
+
+	/**
+	 * query to filter sidebar items. esp. use in pulse sidebar.
+	 * enableFullVodPlaybackInEmbed = true and defaultSidebar should have a value to make it work.
+	 */
+	sidebarFilterQuery?: string;
+	/**
+	 * flag to override account theme to default theme.
+	 */
+	applyDefaultTheme?: boolean;
+
+	/**
+	 * In case of enableFullVodPlaybackInEmbed = true, user can provide a mobile breakpoint size
+	 * so that player and sidebar will render nicely together.
+	 */
+	mobileLayoutBreakPoint?: boolean;
```

Comment:
is this a `number`? If so please include a note about the default value / sane values in the comments